birdhouse/default.env

```diff
@@ -1,6 +1,23 @@
 # All env in this default.env can be overridden by env.local.
 # All env in this default.env must not depend on any env in env.local.
 
+# General metadata details about the instance
+# Details a shared across various containers wherever applicable
+export SERVER_SYNOPSIS="The PAVICS (Power Analytics for Visualization of Climate Science) platform is a collection of climate analysis services served through Open Geospatial Consortium (OGC) protocols. These services include data access, processing and visualization. Both data and algorithms can be accessed either programmatically, through OGC-compliant clients such as QGIS or ArcGIS, or a custom web interface."
+export SERVER_INSTITUTION="Ouranos"
+export CONTACT_NAME="support"
+export CONTACT_POSITION="support"
+export CONTACT_PHONE=""
+export CONTACT_FAX=""
+export CONTACT_DELIVERY_POINT=""
+export CONTACT_CITY="Montréal"
+export CONTACT_STATE_OR_PROVINCE="Québec"
+export CONTACT_COUNTRY="Canada"
+export CONTACT_POSTAL_CODE="H3A 1B9"
```

Comment:
LOL is this Ouranos's postal code?  Maybe put something bogus here, like the bogus support email.

Reply:
Yes it is. Will replace.